tools/coverage.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+
+set -e
+
+echo "[!] Generating coverage report ... "
+pushd build/Debug
+
+mkdir -p coverage/full/html
+llvm-profdata merge -sparse *.profraw -o coverage/full/full.profdata
+llvm-cov export \
+	-format lcov \
+	--instr-profile coverage/full/full.profdata \
+	"blake2b" \
```

Comment:
This way we only get coverage for the binaries we have. But we want to have the coverage for the entire library. So we need to add the library here as well.